crates/cheatnet/src/state.rs

```diff
@@ -320,7 +318,7 @@ pub struct CheatnetState {
     pub mocked_functions:
         HashMap<ContractAddress, HashMap<EntryPointSelector, CheatStatus<Vec<StarkFelt>>>>,
     pub replaced_bytecode_contracts: HashMap<ContractAddress, ClassHash>,
-    pub spies: Vec<SpyTarget>,
+    pub spies: Vec<usize>,
```

Comment:
Why do we need this field? Could it be removed?

Reply:
Yup, forgot that one, thanks!